compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/transform/AbstractDoFnTransform.java

```diff
@@ -281,8 +281,7 @@ public TimerInternals timerInternals() {
     final DoFn wrappedDoFn = wrapDoFn(doFn);
 
     // invoker
-    doFnInvoker = DoFnInvokers.invokerFor(wrappedDoFn);
-    doFnInvoker.invokeSetup();
+    doFnInvoker = DoFnInvokers.tryInvokeSetupFor(wrappedDoFn); // options?
```

Comment:
How about adding an exception handling statement?

Reply:
![image](https://user-images.githubusercontent.com/6691311/182068077-ce4bc5cc-8dfd-4542-a1ae-6d4a610cc29c.png)
As you can see from the source code, the exception is already handled inside the function.